.github/workflows/ruby.yml

```diff
@@ -40,8 +40,11 @@ jobs:
         run:  |
           gem update --system 2.7.10 --no-document
 
+      - name: Update Bundler
+        run: gem install bundler
```

Comment:
Your PR fails on Ruby 2.2 because you're install Bundler 2 on Ruby 2.2. Instead, you need to change the Github action so that you're running `gem install bundler -v 1.17.3` when running Ruby 2.2.

Reply:
The 2.2 Rubies have Bundler 1.7.3 installed already.  See the 2.2 jobs at:
https://github.com/ruby/setup-ruby/runs/491284436